### PR TITLE
Dat 2770 - add different thumbnail size for infobox images on mobile

### PR DESCRIPTION
**Pull request by RafLeszczynski**

### CODE REVIEW

File: extensions/wikia/PortableInfobox/services/PortableInfoboxRenderService.class.php [hunk 1]
```diff
@@ -149,7 +150,11 @@ private function renderItem( $type, array $data ) {
 	}
 
 	protected function getThumbnailUrl( $url ) {
-		return VignetteRequest::fromUrl( $url )->scaleToWidth( self::THUMBNAIL_WIDTH )->url();
+		return VignetteRequest::fromUrl( $url )->scaleToWidth(
+			F::app()->checkSkin( 'wikiamobile' ) ?
```
Comment (SebastianMarzjan) [inline extensions/wikia/PortableInfobox/services/PortableInfoboxRenderService.class.php]: wikiamobile?

Comment (RafLeszczynski) [inline extensions/wikia/PortableInfobox/services/PortableInfoboxRenderService.class.php]: I've talked to @hakubo about it. Our PHP doesn't know about mercury and it's gets the same HTML article content as wikiamobile

**Comment by SebastianMarzjan** [comment]
LG

**Comment by SebastianMarzjan** [comment]
Please fix the tests, then LGTM

**Pull request merged**